src/Domain/Model/Product/ProductTaxa/ProductTaxon.php

```
@@ -0,0 +1,61 @@
+    public function getMappedData(): array
+    {
+        return [
+            'product_id' => $this->productId->get(),
+            'taxonomy_id' => $this->taxonomyId->get(),
+            'taxonomy_type' => $this->taxonomyType->value,
+            'taxon_id' => $this->taxonId->get(),
+            'data' => json_encode($this->data),
+        ];
+    }
+
+    public static function fromMappedData(array $state, array $aggregateState, array $childEntities = []): static
```

PHPMD (2.15.0)

Avoid unused parameters such as '$childEntities'. (Unused Code Rules)

(UnusedFormalParameter)